firebase-ai/src/main/kotlin/com/google/firebase/ai/type/Part.kt

```diff
@@ -53,6 +53,12 @@ internal constructor(
   ) : InternalPart
 }
 
+/**
+ * Represents the code execution result from the model.
+ * @property outcome The result of the execution.
+ * @property output The stdout from the code execution, or an error message if it failed.
+ * @property isThought Indicates whether the response is a thought.
+ */
```

Comment:
This is not a strict change for this PR, but do we really need `CodeExecutionResultPart` to have a public constructor? Is the dev supposed to create one?